woocommerce/woocommerce
# Add `(deprecated)` to Reports menu, move to the last position

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Add `(deprecated)` to the Reports menu and move it to the last position.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

0. Checkout and build this branch
1. Go to `/wp-admin/admin.php?page=wc-admin`
2. Check that Reports menu entry has the `(deprecated)` note added and is the last in the list.
3. Go to `/wp-admin/admin.php?page=wc-reports` Check that the header has the `(deprecated)` note
   ![image](https://github.com/user-attachments/assets/eecf943a-9734-4152-b072-768579cf4718)
4. Log in as a user without `edit_others_shop_orders` capability or
	```php
	add_filter( 'user_has_cap', function( $caps ) {
	    $caps['edit_others_shop_orders'] = false;
	    return $caps;
	} );
	```
5. go to `/wp-admin`
6. Check that `Sales reports` menu entry has `deprecated` note added
	![image](https://github.com/user-attachments/assets/23e44614-5c0c-4695-b9c4-e4edd86c42b9)





### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
